docs/source/basic/intro.rst

```diff
@@ -0,0 +1,90 @@
+This allows to achieve portability of performant codes across various types of accelerators by ignoring specific unsupported levels and utilizing only the ones supported on a specific accelerator.
+All hardware types (multi- and many-core CPUs, GPUs and other accelerators) are treated and can be programmed in the same way.
+The *alpaka* library provides back-ends for *CUDA*, *OpenMP*, *Boost.Fiber* and other methods.
+The policy-based C++ template interface provided allows for straightforward user-defined extension of the library to support other accelerators.
+
+The library name *alpaka* is an acronym standing for **A**\ bstraction **L**\ ibrary for **Pa**\ rallel **K**\ ernel **A**\ cceleration.
+
+Example
+-------
+
+.. literalinclude:: ../../../example/helloWorld/src/helloWorld.cpp
+   :language: C++
+   :caption: helloWorld.cpp
+
+.. code-block:: cmake
+   :caption: CMakeLists.txt
+
+   cmake_minimum_required(VERSION 3.15)
+
+   set(_TARGET_NAME helloWorld)
+   project(${_TARGET_NAME})
+
+   find_package(alpaka REQUIRED)
+
+   alpaka_add_executable(${_TARGET_NAME} helloWorld.cpp)
+   target_link_libraries(
+     ${_TARGET_NAME}
+     PUBLIC alpaka::alpaka)
+
+You can integrate alpaka into your project via ``find_package()`` in your ``CMakeLists.txt``.
+This requires, that you :doc:`install </basic/install>` alpaka.
+If you do not install alpaka in a default path such as ``/usr/local/`` you have to set the ``CMake`` argument ``-Dalpaka_ROOT=/path/to/alpaka/install``.
+
+The cmake configuration decides which ``alpaka::acc`` is available during compiling. For example, if you configure your ``cmake`` build with the CUDA backend (``cmake -DALPAKA_ACC_GPU_CUDA_ENABLE=ON``), ``cmake`` checks, if the CUDA SDK is available and if it found, the C++ template ``alpaka::acc::AccGpuCudaRt`` is available during compiling.
+
+What is alpaka
+--------------
+
+alpaka is ...
+~~~~~~~~~~~~~
+
+A Abstract Interface
+   It describing parallel execution on multiple hierarchy levels. It allows to implement a mapping to various hardware architectures but is no optimal mapping itself.
+
+Sustainably
+   *alpaka* decoupled the application from the availability of different accelerator frameworks in different versions, such as OpenMP, CUDA, HIP, etc. (50% on the way to reach full performance portability).
+
+Heterogeneity
+   An identical algorithm / kernel can be executed on heterogeneous parallel systems by selecting the target device. This allows the best performance for each algorithm and/or a good utilization of the system without major code changes.
+
+Maintainability
+   *alpaka* allows to provide a single version of the algorithm / kernel that can be used by all back-ends. There is no need for "copy and paste" kernels with different API calls for different accelerators. All the accelerator dependent implementation details are hidden within the *alpaka* library.
+
+Testability
```

Comment:
This list is a little weird with a mix of specific nouns (interface), general category nouns (maintainability) and adjectives (optimizable).

Reply:
I agree with you, but I'm not good a finding good headings. This is the reason, why the second part (`alpaka is not ...`) has sentences as headings. Do you have some suggestions?

Reply:
Yes, I could help with phrasing these.

Reply:
Okay, so I think the first five "alpaka is..." items can be reformulated as:

- Abstract
- Sustainable
- Heterogeneous
- Maintainable
- Testable

The last three already have this style and can be left as is

Reply:
We could also use "alpaka features...". Then we could keep the nouns and just rephrase "Sustainably".

Reply:
This is another way. Maybe "alpaka enables..." then?